config/locales/views/en.yml

```diff
@@ -528,6 +528,15 @@ en:
       more_link:
         one: "%{count} more user"
         other: "%{count} more users"
+  languages:
+    index:
+      add: Add a language
+      description: Work Languages
+      edit: Edit
+      suggest: Suggest a language
```

Comment:
Could you keep the original capitalization from the navigation actions? Our navigation should always be title case. (You can, however, drop the "a" if you want.) It also wouldn't hurt to group them under `navigation` like we do elsewhere to give Translation a bit more context, e.g. https://github.com/otwcode/otwarchive/blob/653c19c250d9e35b2ea33e86a51b7b28c1e4c975/config/locales/views/en.yml#L356-L361

Reply:
Sounds good! I think keeping the `a` sounds better but I can remove it if that's the standard